.travis.yml

```diff
@@ -4,17 +4,30 @@ go_import_path: github.com/KohlsTechnology/eunomia
 go:
 - "1.12"
 env:
-  - GO111MODULE=on
+  - GO111MODULE=on CHANGE_MINIKUBE_NONE_USER=true
 
 before_install:
   - travis_retry go mod vendor
   - go get golang.org/x/lint/golint
 
+before_script:
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.14.2/bin/linux/amd64/kubectl
+- sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+- sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
+- curl -LO https://git.io/get_helm.sh && chmod 700 get_helm.sh && ./get_helm.sh
+- RELEASE_VERSION=v0.9.0; curl -OJL https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk && rm operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
```

Comment:
Is there a more central place we can define versions in? Would make it easier for future maintenance.

Reply:
@Smiley73 done.

Reply:
@etsauer I do not see the changes...